spec/models/logstash_queries/elastic_link_popularity_query_spec.rb

```diff
@@ -4,23 +4,28 @@
   let(:query) { ElasticLinkPopularityQuery.new('https://search.gov', 10) }
   let(:expected_body) do
     {
-      query: {
-        constant_score: {
-          filter: {
-            bool: {
-              must: [
+      "query": {
+        "constant_score": {
+          "filter": {
+            "bool": {
+              "must": [
                 {
-                  terms: {
-                    'params.url': [
-                      'https://search.gov',
-                      'https://search.gov/'
+                  "term": {
+                    "type": "click"
+                  }
+                },
```

Comment:
I updated the query JSON to include the new `type` filter. I also converted the test hash to use double-quotes instead of single quotes. We've done that throughout the logstash query specs, as it makes it possible to copy & paste the query into Kibana for testing.